apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -139,6 +139,25 @@ handle_request('GetAccountBalance', Req, _Context) ->
             end
     end;
 
+handle_request('GetName', Req, _Context) ->
+    Name = case maps:get('name', Req) of
+               undefined ->
```

Comment:
Would it be possible to enrich the [Swagger spec query parameter](https://github.com/aeternity/epoch/pull/607/files#diff-89b8ec7c2f28f3e3c7d21ca9bf070bb7R223) with property [`required` true](https://github.com/OAI/OpenAPI-Specification/blob/55a4c6d8498b24faa8529b386f04bfb47a3ecfa5/versions/2.0.md#fixed-fields-7), and hardcode expectation here on `maps:get/2` to always return binary - relying on Swagger codegen to return 400? Cc @velzevur @ThomasArts